app/static/src/app/store/graphs/state.ts

```diff
@@ -0,0 +1,17 @@
+export type YAxisRange = [number, number];
+
+export interface GraphSettings {
+    logScaleYAxis: boolean;
+    lockYAxis: boolean;
+    yAxisRange: YAxisRange;
+}
+
+export interface GraphConfig {
```

Comment:
im assumng config will have more than just selected and unselected variables soon?

Reply:
Yeah, will have settings (e.g log scale, y axis range. Maybe label  too eventually.